python_modules/dagster-graphql/dagster_graphql/schema/asset_graph.py

```diff
@@ -546,23 +550,63 @@ def get_materialization_status_for_asset_health(self, graphene_info: ResolveInfo
 
         return GrapheneAssetHealthStatus.UNKNOWN
 
-    def get_asset_check_status_for_asset_health(self, graphene_info: ResolveInfo):
-        # all asset checks passed, health
-        # any asset check warning, warning
-        # any asset check failed, degraded
-        return GrapheneAssetHealthStatus.UNKNOWN
+    async def get_asset_check_status_for_asset_health(self, graphene_info: ResolveInfo):
+        remote_check_nodes = graphene_info.context.asset_graph.get_checks_for_asset(
+            self._asset_node_snap.asset_key
+        )
+        if not remote_check_nodes:
+            return GrapheneAssetHealthStatus.UNKNOWN
```

Comment:
I wonder if we need a new status like `GrapheneAssetHealthStatus.NOT_APPLICABLE` to capture a case like this

Reply:
+1. If the user hasn't implemented checks, but their SLAs + materialization health is good, I would expect their overall status to be healthy.